src/plugins/data/common/search/search_source/search_source.ts

```diff
@@ -439,7 +440,12 @@ export class SearchSource {
       getConfig,
     });
 
-    return search({ params, indexType: searchRequest.indexType }, options).pipe(
+    // force a rollup strategy in case this is a rollup index
+    if (!options.strategy && searchRequest.indexType === 'rollup') {
```

Comment:
I think this should go together with the code that handles `syncSearchByDefault`.

I think both of those things would fit nicely in line 290, where we set the `indexPattern`

Reply:
Nice. didn't notice this 👍